docs/feature-flagging/01-feature-gates.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
```

Comment:
nit, these pages are now numbered `01-feature-gates.md` etc, but the sidebar position got shifted